CHANGELOG.md

```diff
@@ -4,8 +4,25 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### [v6.139.0](https://github.com/opengovsg/FormSG/compare/v6.139.0...v6.139.0)
+
+- fix: forbid stripe acc connection with pdf summary enabled [`#7570`](https://github.com/opengovsg/FormSG/pull/7570)
+- fix: failing gh actions due to runner out of disk space  [`#7565`](https://github.com/opengovsg/FormSG/pull/7565)
+
+#### [v6.139.0](https://github.com/opengovsg/FormSG/compare/v6.138.0...v6.139.0)
```

Comment:
nit: could combine these two?

Reply:
Best that this is re-checked when we cut a new release. Not 100% sure that this wouldn't be re-generated by the autochangelog script.